### ISSUE TITLE
Bug in error handling

### ISSUE THREAD

**Issue by Kuzmin** [burnash/gspread#311]
When the Google API sometimes return errors, the gspread library throws the following Exception

```
Traceback (most recent call last):
  File "COGS-tools/main.py", line 114, in <module>
    conn.ri_uploader.upload(data, sheet_name, i['w_sheet'])
  File "/home/ubuntu/COGS-tools/make_sheet.py", line 60, in upload
    worksheet.update_cell(row_index, column_index, data)
  File "/usr/local/lib/python2.7/dist-packages/gspread/models.py", line 445, in update_cell
    self.client.put_feed(uri, ElementTree.tostring(feed))
  File "/usr/local/lib/python2.7/dist-packages/gspread/client.py", line 282, in put_feed
    if ex.code == 403:
AttributeError: 'HTTPError' object has no attribute 'code'
```

**Comment by msuozzo** [comment]
This has been fixed in commit f3cb9f8b6142a7d7fb07acce3fb33370c5403802. It will be live in the next release.

**Comment by Kuzmin** [comment]
Left a comment on the commit.

Thanks for the great job guys!
